requirements.txt

```diff
@@ -2,6 +2,7 @@ rdkit
 numpy
 networkx
 matplotlib
+pillow=9.5.0
```

Comment:
Why do you need pillow? Does it have something to do with mamba? If so, would it be better to use pillow>=9.50?

Reply:
pillow is a [matplotlib dep](https://matplotlib.org/stable/devel/dependencies.html) which I pinned to fix the broken dependency installs on the CI: https://github.com/duartegroup/autodE/actions/runs/5432398647/jobs/9879412904#step:5:31 
probably smart to go for `>=`, if it works! 🤞🏼

Reply:
looks like it does 🎉  thanks for the suggestion